.github/workflows/rust.yml

```diff
@@ -271,8 +275,8 @@ jobs:
                   # `listen-addr` is used to configure p2p to accept websocket connections instead of TCP.
                   # `node-key` provides a deterministic p2p address.
                   substrate --dev --node-key 0000000000000000000000000000000000000000000000000000000000000001 --listen-addr /ip4/0.0.0.0/tcp/30333/ws > /dev/null 2>&1 &
-                  wasm-pack test --headless --firefox
-                  wasm-pack test --headless --chrome
+                  wasm-pack test --no-default-features --features jsonrpsee,web --headless  --firefox 
+                  wasm-pack test --no-default-features --features jsonrpsee,web --headless --chrome 
```

Comment:
These should be unnecessary shouldn't they? The `wasm-rpc-tests` crate doesn't expose any features, and so this should have no effect.